data/data/gcp/main.tf

```diff
@@ -16,7 +16,7 @@ module "bootstrap" {
 
   bootstrap_enabled = var.gcp_bootstrap_enabled
 
-  image_name   = var.gcp_image_id
+  image        = "${google_compute_image.cluster.self_link}"
```

Comment:
this should follow terraform 0.12 syntax. `google_compute_image.cluster.self_link`